packages/dev/loaders/src/SPLAT/splatFileLoader.ts

```diff
@@ -76,15 +79,21 @@ export class SPLATFileLoader implements ISceneLoaderPluginAsync, ISceneLoaderPlu
 
     /**
      * Creates loader for gaussian splatting files
+     * @param loadingOptions options for loading and parsing splat and PLY files.
      */
-    constructor() {}
+    constructor(loadingOptions?: Partial<Readonly<SPLATLoadingOptions>>) {
+        this._loadingOptions = loadingOptions || SPLATFileLoader._DefaultLoadingOptions;
+    }
 
-    /**
-     * Instantiates a gaussian splatting file loader plugin.
-     * @returns the created plugin
-     */
-    createPlugin(): ISceneLoaderPluginAsync | ISceneLoaderPlugin {
-        return new SPLATFileLoader();
+    private static get _DefaultLoadingOptions(): SPLATLoadingOptions {
+        return {
+            keepInRam: false,
+        };
+    }

```

Comment:
Is there any reason for this to be a property that constructs a new object each time, especially if it is private? Could it be:
```suggestion
    private static readonly _DefaultLoadingOptions = {
        keepInRam: false,
    } as const satisfies SPLATLoadingOptions;
```

Reply:
LGTM